provision-osx/defaults/main.yml

```diff
@@ -43,4 +43,7 @@ homebrew_taps:
 xcode_install_user:
 xcode_install_password:
 xcode_versions:
-- 8
\ No newline at end of file
+- 8
+
+apple_wwdr_cert_url: http://developer.apple.com/certificationauthority/AppleWWDRCA.cer
```

Comment:
I think both can be overridden but users don't have to. These are default values.